code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -728,6 +735,16 @@
 				m_type = 1
 				playsound(src.loc, 'sound/voice/joe_awful.ogg', 75, 0)
 				say("Tut, tut. What an awful mess.")
+		if("mess")
+			if(Joe && src.loc)
+				m_type = 1
+				playsound(src.loc, 'sound/voice/joe_awful.ogg', 75, 0)
+				say("Tut, tut. What an awful mess.")
+		if("mes") //Since the parser trims the final s...
+			if(Joe && src.loc)
+				m_type = 1
+				playsound(src.loc, 'sound/voice/joe_awful.ogg', 75, 0)
+				say("Tut, tut. What an awful mess.")
```

Comment:
This is really weird but if you absolutely need this, you can use `if("mess","mes")`

Reply:
Ah neat, thanks for the tip, fixed!